specs/nest_cli/generate_app.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Generate an app
```

Comment:
Am realizing that it might be confusing to people what these workflows pertain to from the in-app search. Can we add nest.js to the workflow names so people don't get confused.

Reply:
Maybe something like:
"Generate an app using Nest.js"